packages/uiweb/src/lib/components/chat/ChatProfile/ChatProfile.tsx

```diff
@@ -1,182 +1,222 @@
-        const result = await resolveNewEns(chatId, provider);
-        // if(result) 
-        setEnsName(result);
-      }
+  };
+
+  const getImage = () => {
+    if (chatInfo || groupInfo) {
+      return isGroup
+        ? groupInfo?.groupImage ?? GreyImage
+        : chatInfo?.profile?.picture ??
+            createBlockie?.(chatId)?.toDataURL()?.toString();
+    } else {
+      return createBlockie?.(chatId)?.toDataURL()?.toString();
     }
-
-
-    useEffect(()=> {
-        if(!chatId) return;
-        fetchProfileData();
-        getName(chatId);
-    },[chatId, account, env])
-
-    if (chatId && style === 'Info') {
-        return (
-            <Container theme={theme}>
-                {chatInfo || groupInfo ? (
-                    <Image src={isGroup ? groupInfo?.groupImage ?? GreyImage : chatInfo?.profile?.picture ?? createBlockie?.(chatId)?.toDataURL()
-            ?.toString()} height="48px" maxHeight="48px" width='48px' borderRadius="100%" />
-                ) : (<Image src={createBlockie?.(chatId)?.toDataURL()
-                    ?.toString()} height="48px" maxHeight="48px" width='48px' borderRadius="100%" />)}
-                
-
-                <Span color={theme.textColor?.chatProfileText} fontSize="17px" margin="0 0 0 10px">
-                    {isGroup ? groupInfo?.groupName : ensName ? `${ensName} (${isMobile ? shortenText(chatInfo?.did?.split(':')[1] ?? '', 4, true) : chatId})`: chatInfo ? shortenText(chatInfo.did?.split(':')[1] ?? '', 6, true) : shortenText(chatId,6, true)}
-                
-                </Span>
-
-                <Options 
-                    options={options} 
-                    setOptions={setOptions} 
-                    isGroup={isGroup} 
-                    chatInfo={chatInfo} 
-                    groupInfo={groupInfo} 
-                    setGroupInfo={setGroupInfo} 
-                    theme={theme} 
-                    />
-
-                {/* {!isGroup && 
+  };
+
+  const getProfileName = () => {
+    return isGroup
+      ? groupInfo?.groupName
+      : ensName
+      ? `${ensName} (${
+          isMobile
+            ? shortenText(chatInfo?.did?.split(':')[1] ?? '', 4, true)
+            : chatId
+        })`
+      : chatInfo
+      ? shortenText(chatInfo.did?.split(':')[1] ?? '', 6, true)
+      : shortenText(chatId, 6, true);
+  };
+
+  useEffect(() => {
+    if (!chatId) return;
+    fetchProfileData();
+  }, [chatId, account, env]);
+
+  if (chatId && style === 'Info') {
+    return (
+      <Container theme={theme}>
+        <ProfileContainer
+          theme={theme}
+          member={{ wallet: getProfileName() as string, image: getImage() }}
+          customStyle={{fontSize:'17px'}}
+        />
+
+        <Options
+          options={options}
+          setOptions={setOptions}
+          isGroup={isGroup}
+          groupInfo={groupInfo}
+          setGroupInfo={setGroupInfo}
+          theme={theme}
+        />
+
+        {/* {!isGroup && 
```

Comment:
are we leaving video call options for wallet 2 wallet conversation right now ?

Reply:
yess we are